### PR TITLE
Feat/avatar and badge

### PR DESCRIPTION
**Pull request by tonymike99**
I have added the avatar and badge components.

### CODE REVIEW

File: assets/css/components.css [hunk 1]
```diff
@@ -0,0 +1,162 @@
+@import url('../../Components/Avatar/avatar.css');
+@import url('../../Components/Badge/badge.css');
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;900&display=swap');
+
+* {
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+    /* font-size: 62.5%; */
```
Comment (zoef-yt) [inline assets/css/components.css:9]: why is this code commented?

**Review by zoef-yt — APPROVED** [review]
Looks good to me.

File: assets/css/components.css (hunk 1 above)
Comment (tonymike99) [reply assets/css/components.css:9]: I initially tried it out, Zoef. But then every css element looked very small. So I commented it for the time being.

**Review by sanu29 — APPROVED** [review]
_Looks great for me. Resetting done great. Declarations done in one file (colors and utility classes) which is good.  File structure is understandable. Spacing could be better in some places and extra comments could be removed. 
Rest all good no major changes that could be suggested._

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    display: inline-block;
+    position: relative;
+    margin: 1rem;
+}
+
+.badge {
+    background-color: var(--light-color);
+    border-radius: 5px;
+    font-size: 10px;
+    font-weight: bold;
```
Comment (dev-prabhat) [inline Components/Badge/badge.css:15]: Try to give length or size in relative units like rem or em

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Try to use relative units instead of absolute like rem or em

**Review by iRohitGaur — APPROVED** [review]
Looks good 👍

File: Components/Badge/badge.css (hunk 2 above)
Comment (tonymike99) [reply Components/Badge/badge.css:15]: Thanks Prabhat for the review! But I guess Vikram suggested not to use 'rem' for 10px and lesser.

**Pull request merged**